app/views/unions/index.html.haml

```diff
@@ -12,9 +12,6 @@
       %td
         = union.name
       %td{class: 'min-width', align: 'center'}
-        = form_tag union, method: :delete, data: { confirm: 'Are you sure?' } do
-          = button_tag(class: 'btn btn-xs btn-danger') do
-            %span.glyphicon.glyphicon-trash
-            Delete
-
-
+        = button_to union_path(union), class: 'btn btn-xs btn-danger' do
```

Comment:
This doesn't work, it's no longer a delete request and you removed the confirmation.

Reply:
Also it errors.

Reply:
But it looks nice though!